idd/Energy+.idd.in

```diff
@@ -45619,7 +45615,131 @@ Coil:Heating:DX:VariableSpeed,
+       \required-field
+       \type alpha
+  A5, \field Condenser Water Inlet Node
+       \required-field
+       \type alpha
+  A6, \field Condenser Water Outlet Node
+       \required-field
+       \type alpha
+  A7, \field Water Node to Monitor the Supply Hot Water Flow
+       \required-field
+       \type alpha
+  A8, \field Outdoor Air Inlet Node
+       \required-field
+       \type alpha
+  A9, \field Outdoor Air Outlet Node
+       \required-field
+       \type alpha
+  A10, \ field Coil Name of Space Cooling Mode
+       \required-field
+       \type object-list
+       \object-list CoolingCoilsDXVariableSpeed
+       \note Must match the name used in the corresponding Coil:Cooling:DX:VariableSpeed object.
+  A11,  \field Coil Name of Space Heating Mode
+       \type object-list
+       \object-list HeatingCoilsDXVariableSpeed
+       \note Must match the name used in the corresponding Coil:Heating:DX:VariableSpeed object.
+  A12,  \field Coil Name of Dedicated Water Heating Mode
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.
+  A13, \field Coil Name of Combined Space Cooling and Water Heating Mode with Full Condensing
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.
+  A14, \field Cooling Coil Name of Combined Space Cooling and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list CoolingCoilsDXVariableSpeed
+       \note Must match the name used in the corresponding Coil:Cooling:DX:VariableSpeed object.
+  A15, \field Water Heating Coil Name of Combined Space Cooling and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.	   
+  A16, \field Heating Coil Name of Combined Space Heating and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list HeatingCoilsDXVariableSpeed
+	   \note Must match the name used in the corresponding Coil:Heating:DX:VariableSpeed object.   
+  A17, \field Water Heating Coil Name of Combined Space Heating and Water Heating Mode with Desuperheating
+       \type object-list
+       \object-list HeatPumpWaterHeaterDXCoils
+       \note Must match the name used in the corresponding Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed object.	   	   
+  N1 , \field Indoor Temperature Limit for SCWH Mode
+       \type real
+       \units C
+       \minimum> 15.0
+       \default 20.0
+       \note Indoor Temperature above which Indoor Overcooling is Allowed during Cooling Operation
+  N2 , \field Ambient Temperature Limit for SCWH Mode
+       \type real
+       \units C
+       \minimum>20.0
```

Comment:
I'm not sure if IP will have trouble with a missing space here, but I'd go ahead and add a space after the >